vnext/Shared/OInstance.cpp

```diff
@@ -395,15 +395,15 @@ InstanceImpl::InstanceImpl(
 #if defined(USE_V8)
           std::unique_ptr<facebook::jsi::PreparedScriptStore> preparedScriptStore;
 
-          char tempPath[MAX_PATH];
-          if (GetTempPathA(MAX_PATH, tempPath)) {
-            preparedScriptStore = std::make_unique<facebook::react::BasePreparedScriptStoreImpl>(tempPath);
+          wchar_t tempPath[MAX_PATH];
+          if (GetTempPathW(MAX_PATH, tempPath)) {
```

Comment:
@asklar I'm getting local build errors complaining about the identifier not being found. I even tried explicitly including `Windows.h`.
Perhaps some of the pre-processor definitions leave this API out?
I've triggered a build to see if this happens in the UWP variant.

Reply:
it looks like GetTempPath2 is only available in the Fe SDK, so keep the GetTempPath usage, just use the std::size instead so if we ever think of changing the buffer size, we don't accidentally introduce a buffer overrun because the api call uses MAX_PATH directly.

Reply:
Got it. Its presence in the current SDK ( < 20348 ) headers deceived me.